plugins/lookup/conjur_variable.py

```diff
@@ -203,12 +210,24 @@ def _fetch_conjur_variable(conjur_variable, token, conjur_url, account, validate
     return {}
 
 
+def _store_in_file(value):
+    file_name = ''.join(choice(string.ascii_uppercase + string.digits) for i in range(12))
```

Comment:
You probably should use something like [`tempfile.NamedTemporaryFile`](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile) and set prefix/suffix/dir instead of manually assembling the name.

Reply:
Thanks, I didn't know this lib. I made the change in next commit